docs/docs/new-features.md

```
@@ -187,15 +187,15 @@ Feature parity with the current media framework is not yet achieved, but the
 
 To enable support for LiveKit:
-  - Install bbb-livekit: `$ sudo apt-get install bbb-livekit`
-  - Enable the LiveKit controller module in bbb-webrtc-sfu: `$ sudo yq e -i '.livekit.enabled = true' /etc/bigbluebutton/bbb-webrtc-sfu/production.yml`
-  - Restart bbb-webrtc-sfu: `$ sudo systemctl restart bbb-webrtc-sfu`
-  - Guarantee that Node.js 22 is installed in your server: `$ node -v`
-    - Older 3.0 installations might still be using Node.js 18. If that's the case,
+1. Install bbb-livekit: `$ sudo apt-get install bbb-livekit`
+2. Enable the LiveKit controller module in bbb-webrtc-sfu: `$ sudo yq e -i '.livekit.enabled = true' /etc/bigbluebutton/bbb-webrtc-sfu/production.yml`
+3. Restart bbb-webrtc-sfu: `$ sudo systemctl restart bbb-webrtc-sfu`
+4. Guarantee that Node.js 22 is installed in your server: `$ node -v`
+    * Older 3.0 installations might still be using Node.js 18. If that's the case,
```

Markdownlint (0.37.0)

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)